handlers/search.go

```diff
@@ -69,11 +69,18 @@ func SearchAhead(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
 	searchType := vars["type"]
 	search := vars["search"]
-	search = strings.Replace(search, "0x", "", -1)
```

Comment:
You have to use `strippedSearch` here now:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/26c2b9cae9ee4a2cd1a9732c36afa06e796927f6/handlers/search.go#L226
as the `0x` prefix is not removed from `search` anymore but the RE does not accept it.

Reply:
changed